src/main/java/io/fabric8/maven/docker/AbstractDockerMojo.java

```diff
@@ -140,6 +140,12 @@
     @Parameter(property = "docker.registry")
     protected String registry;
 
+    /**
+     * Skip exchanging local iam credentials for ecr credentials
+     */
+    @Parameter(property = "docker.exchange.skip")
```

Comment:
I would call it something related to ECR as this feature is connected to the way how AWS does authentication (or is this useful in another context, too ? Mayeb GCR does something similar, but haven't checked).

Also the common schem is `docker.skip.something`, so for the property name I'd suggest

```java
@Parameter(property = docker.skip.ecrAuth)
protected boolean skipEcrAuth;
```